docs/pages/index.mdx

```diff
@@ -6,102 +6,72 @@ layout: tocless-doc
 videoBanner: fSgCnQrNu78
 ---
 
-Teleport is a certificate authority and access plane for your infrastructure.
-With Teleport you can:
+Teleport is Identity-Native Infrastructure Access for engineers and machines. With Teleport you can:
```

Comment:
Just engineers and machines? I've been hearing feedback that we need to keep in mind the end user of services behind Teleport, which can include applications accessed by non-engineers.

Reply:
I'm sticking to the messaging guidelines as written for now.  If those evolve so too can this. I'll take suggestions though if there's a better way to phrase this.